server/wfprev-api/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.3.3</version>
+		<version>3.2.1</version>
```

Comment:
Do we want to downgrade? Should be sticking with latest non-snapshot if possible

Reply:
As discussed, I was getting issues from the ministry artifactory.  it doesn't appear they have this version availble.